config/environments/development.rb

```diff
@@ -10,7 +10,7 @@
 
   # Do not eager load code on boot.
```

Comment:
Should probably update the comment here to describe why we're setting this to true.